packages/react/src/components/UIShell/UIShell-story.js

```diff
@@ -364,7 +319,93 @@ storiesOf('[Experimental] UI Shell', module)
     ))
   )
   .add(
-    'SideNav',
+    'Fixed SideNav',
+    withReadme(readme, () => (
+      <>
+        <SideNav isFixedNav expanded={true} aria-label="Side navigation">
```

Comment:
I added a prop to `SideNav` called `isChildOfHeader`, which will accept a bool. This adds specific styles for the header/sidenav ux. I set it as true by default (we can change that if we want), but the example for fixed `SideNav` is lower by 48px since it doesn't have `isChildOfHeader={false}` so it's adding the header-ux styles.

Reply:
Updated 👍